conda/cli/helpers.py

```diff
@@ -0,0 +1,501 @@
+# Copyright (C) 2012 Anaconda, Inc
```

Comment:
Does it make more sense to put these in `conda.cli.common`. I am curious why a new module was created.